### PR TITLE
Added information to the readme about building and running.

### PR DESCRIPTION
**Pull request by elistone**
Small tidy of the readme, plus adding information about how to run and build the application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,68 @@ Each fact is determined by a "gatherer", essentially an object that implements t
...
+$ go run main.go [command] 
+$ go run main.go gather # run the gather command
+```
+
+However, most normal circumstances this will fail to run, as the gather will try to look for environment variables.
+
+You can preface the command with these environment variables, this will allow it to continue.
+
+```bash
+$ LAGOON_PROJECT=[proejct_name] LAGOON_GIT_BRANCH=[git_branch] go run main.go gather
```
Comment (timclifford) [inline README.md:45]: typo

File: README.md [hunk 2]
```diff
@@ -7,9 +7,68 @@ Each fact is determined by a "gatherer", essentially an object that implements t
...
+```
+
+Although you will be doing a scan of your local machine, and will send this information as facts up to which ever project has been set.
+
+**Not recommend** but might be required for some testing.
```
Comment (timclifford) [inline README.md:50]: *recommended

File: README.md (hunk 1 above)
Comment (elistone) [reply README.md]: Made the change

File: README.md (hunk 2 above)
Comment (elistone) [reply README.md]: Made the change

**Pull request merged**